src/auth/jwtService.ts

```diff
@@ -0,0 +1,41 @@
+import * as jwt from 'jsonwebtoken';
+import { Exception } from '../exception';
+import { injectJWTConfig } from './injections';
+import { IJWTConfig } from './jwtConfig';
+
+export class jwtService {
```

Comment:
1. we name classes with PascalCase so `jwtService` should be `JWTService` (as you called it `JWT` everywhere else)